pipelines/azure-pipelines.publish.yml

```diff
@@ -133,11 +133,15 @@ extends:
                 *.psd1
 
           steps:
-          - pwsh: |
```

Comment:
You could just replace `pwsh` by `powershell` here. Both are just an alias for `PowerShell@2` but with different values for the `pwsh` input

Reply:
Oh didn't know that! Will keep this in mind for the future 😊